### PR TITLE
hardcode netns struct offset to 48 bytes 

### PR DESCRIPTION
**Pull request by akarpz**
all kernels I tested had a netns offset of 48 bytes so there isn't a reason to guess for it. This may help to resolve some flakyness in guessing we have seen in conntrack as well, as it relies on a properly guessed netns inode offset to function properly.
research: https://docs.google.com/spreadsheets/d/1Sug6g4VbnwNJhSmAKeIdqHpGZpQY9oH6DOJnF7hebG8/edit#gid=0

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/network/tracer/offsetguess/tracer.go [hunk 1]
```diff
@@ -728,9 +737,10 @@ func (t *tracerOffsetGuesser) Guess(cfg *config.Config) ([]manager.ConstantEdito
 	t.guessUDPv6 = cfg.CollectUDPv6Conns
 	t.guessTCPv6 = cfg.CollectTCPv6Conns
 	t.status = &TracerStatus{
-		State: uint64(StateChecking),
-		Proc:  Proc{Comm: cProcName},
-		What:  uint64(GuessSAddr),
+		State:        uint64(StateChecking),
+		Proc:         Proc{Comm: cProcName},
+		What:         uint64(GuessSAddr),
+		Offset_netns: 48,
```
Comment (hmahmood) [inline pkg/network/tracer/offsetguess/tracer.go:743]: Can we define a `const` for this?

File: pkg/network/tracer/offsetguess/tracer.go [hunk 2]
```diff
@@ -528,23 +529,32 @@ func (t *tracerOffsetGuesser) checkAndUpdateCurrentOffset(mp *ebpf.Map, expected
...
 			t.status.Offset_ino = 0
-			t.status.Offset_netns++
+			// if we have already seen a failure, we need to increment the offset_netns otherwise set it to offset_family
+			if t.status.Seen_failure != 0 {
+				t.status.Offset_netns++
+			} else {
+				t.status.Offset_netns = t.status.Offset_family
+				t.status.Seen_failure = 1
+			}
+			t.status.Offset_netns = t.status.Offset_family
```
Comment (hmahmood) [inline pkg/network/tracer/offsetguess/tracer.go:557]: This shouldn't be here?

Comment (akarpz) [reply pkg/network/tracer/offsetguess/tracer.go]: how did that get there

**Review by hmahmood — APPROVED** [review]

**Pull request merged**